### ISSUE TITLE
The errors.log file is getting filled with Invoke-RestMethod errors on Managed Instances

### ISSUE THREAD

**Issue by austoonz** [aws/amazon-ssm-agent#303]
I have hybrid Managed Instances that obviously are not running the EC2 IMDS. This line is causing the errors.log file to be filled with errors.

This call should have some exception handling so it doesn't throw exceptions when running the SSM Agent on hybrid enabled Windows machines.

https://github.com/aws/amazon-ssm-agent/blob/1074fcbea7002a52aa6312f77baace80ecd7dd62/agent/plugins/inventory/gatherers/billinginfo/dataProvider_windows.go#L38

error.log content:

```text
2020-08-17 23:00:36 ERROR [CollectBillingInfoData @ dataProvider_windows.go.80] [ssm-document-worker] [23790487-d68f-4bab-a6ae-146a22e2f1ba.2020-08-18T06-00-00.618Z] [DataBackend] [pluginName=aws:softwareInventory] [aws:softwareInventory] Command failed with error: Invoke-RestMethod : Unable to connect to the remote server
At line:1 char:1
+ Invoke-RestMethod -uri  http://169.254.169.254/latest/dynamic/instanc ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
```

**Comment by danr-amz** [comment]
Thanks for bringing this to our attention.  We are aware of this issue and are working on a fix.

**Comment by ferkhat-aws** [comment]
This issue has been fixed since version 3.0.196.0.
